resources/upgrading-to-aep/manual-lifecycle-implementation.md

```diff
@@ -0,0 +1,280 @@
+# Manual Lifecycle Implementation in the Experience Platform SDKs
+
+In the Experience Platform SDK, all Lifecycle implementation must be done manually. This page provides a suggested implementation of Lifecycle. The correct implementation for Lifecycle should ultimately be determined by the app developer.
+
+{% hint style="danger" %}
+In version 4 of the iOS SDK, this implementation was done for your automatically.
```

Comment:
Please reword as follows:

"In version 4 of the iOS SDK, this implementation was completed automatically."